recipes/pthreads4w/all/conanfile.py

```diff
@@ -9,6 +9,7 @@ class Pthreads4WConan(ConanFile):
     homepage = "https://sourceforge.net/projects/pthreads4w/"
     description = "POSIX Threads for Windows"
     license = "Apache-2.0"
+    topics = "pthreads"
```

Comment:
c3i will fail because it only allows one recipe change per pr.
This change is in pthread4w.
So you'll have to create a new pr for pthreads4w.

Reply:
Removed.